nodejs/scripts/otel-handler

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export NODE_OPTIONS="--require /opt/adot-extension.js ${NODE_OPTIONS}"
```

Comment:
If we only need to customize env in ADOT for python, I think this delegation approach can work instead of replacing `otel-instrument` as we currently have. Maybe less maintenance